### PR TITLE
Updated update.md

### PR DESCRIPTION
**Pull request by dreamwhite**
Linked issue: [dortania/bugtracker 159](https://github.com/dortania/bugtracker/issues/159)
Please note that the screenshot I added is just a placeholder and needs to be changed.
Also, I feel like that many users can be lead to confusion while seeing the step 3 image, where you highlight the drivers list: please update it accordingly to Configuration.pdf

### CODE REVIEW

File: universal/update.md [hunk 1]
```diff
@@ -51,12 +51,11 @@ So the process goes as follows:
 
 ![](../images/post-install/update-md/oc-config-compare.png)
 
-* Once you've made the adjustments, to make sure that you config is compliant with the newest release of OpenCore, use OpenCore Utility ocvalidate: it'll tell you whether there are errors in your config.
-    * Please note, that `ocvalidate` must match the used OpenCore release and may not be able to detect all configuration flaws present in the file.
-If you want to get a bit more technical read the [Differences.pdf](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Differences/Differences.pdf).
+* Once you've made the adjustments, to make sure that you config is compliant with the newest release of OpenCore, you can the OpenCore Utility ocvalidate: this tool will help ensure your config.plist is matching the OpenCore specification of the matching build.
```
Comment (dreamwhite) [inline universal/update.md:54]: @khronokernel I think that you missed a verb:

* Once you've made the adjustments, to make sure that you config is compliant with the newest release of OpenCore, you can **use** the OpenCore Utility ocvalidate: this tool will help ensure your config.plist is matching the OpenCore specification of the matching build.

File: universal/update.md [hunk 2]
```diff
@@ -51,12 +51,11 @@ So the process goes as follows:
...
+  * Please note, that `ocvalidate` must match the used OpenCore release and may not be able to detect all configuration flaws present in the file. We recommend to double check your setting with the OpenCore Guide on what to set everything to, otherwise read the [Differences.pdf](https://github.com/acidanthera/OpenCorePkg/blob/master/Docs/Differences/Differences.pdf)  for more in-depth documentation on changes.
+  * To run `ocvalidate`, `cd` into OpenCore's `Utilties/ocvalidate/` and run `./ocvalidate <insert_config.plist>`. Note you may need to run `chmod +x ocvalidate` for it to execute.
```
Comment (dreamwhite) [inline universal/update.md:56]: Regarding line 56, probably there's no need to give it the executable octal permission (100).
From what my """experience""" can tell, I never had the need to give it proper permissions, and it always worked.

**Review by dreamwhite — COMMENTED** [review]
universal/update.md small adjustments

File: universal/update.md (hunk 2 above)
Comment (khronokernel) [reply universal/update.md:56]: > From what my """experience""" can tell, I never had the need to give it proper permissions, and it always worked.

Huh, I just ran the program and it refused. Most likely you have given developer tools policy to terminal under Security and Privacy in preferences. By default, most users will have this disabled so best to mention

File: universal/update.md (hunk 1 above)
Comment (khronokernel) [reply universal/update.md:54]: Ah thank you, will push that fix to master in a sec

File: universal/update.md (hunk 2 above)
Comment (dreamwhite) [reply universal/update.md:56]: Uhm, probably you're right. I have disabled GateKeeper with `sudo spctl --master-disable` and never had issues.
At this point, I agree with you that it's best to mention that

**Pull request merged**